src/services/HttpService.js

```diff
@@ -4,7 +4,7 @@ const local = "http://localhost:8080/v1";
 const test = "https://task-management-api-test-5ef9ebfe6fb3.herokuapp.com/v1";
 
 const http = axios.create({
-  baseURL: test,
+  baseURL: local,
```

Comment:
The UI should remain as `test` when being committed. Only change to `local` for local development.